.github/workflows/apt-packages.yaml

```diff
@@ -33,9 +33,8 @@ jobs:
     - name: Add repositories
       run: |
         apt-get update
-        apt-get install -y wget lsb-release gnupg apt-transport-https sudo
-        echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -c -s)-pgdg main" > /etc/apt/sources.list.d/pgdg.list
-        wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+        apt-get install -y wget lsb-release gnupg apt-transport-https sudo postgresql-common
+        yes | /usr/share/postgresql-common/pgdg/apt.postgresql.org.sh
```

Comment:
```suggestion
        /usr/share/postgresql-common/pgdg/apt.postgresql.org.sh -y
```

Reply:
this does not work across all supported versions